.gitignore

```diff
@@ -1,5 +1,10 @@
 bin/
+.gradle
+/local.properties
+/.idea/workspace.xml
```

Comment:
The entire .idea folder should be ignored and we don't want to check in any of the files from there (looks like there's a bunch on your pull request)